CONTRIBUTING.md

```diff
@@ -476,12 +475,13 @@ The category label describes the category.
  - **E-needs-mvce** This bug is missing a minimal complete and verifiable
    example.
 
-(I don't know why it's called `E-`. Many issues are missing a difficulty rating,
-and you should feel free to add one.)
+The "E-" prefix is the same as used in the Rust compiler repository. Some
+issues are missing a difficulty rating, but feel free to ask on our Discord
+server if you want to know how difficult an issue likely is.
 
 ### Module
 
-A more fine grain categorization than **Area**.
+The module label provides a more fine grain categorization than **Area**.
```

Comment:
I'm pretty sure it was me who made this typo originally :sweat_smile: 
```suggestion
The module label provides a more fine grained categorization than **Area**.
```

Reply:
You don't think mine is a better typo?!?